src/repos/volunteerRequestRepo.ts

```diff
@@ -1,16 +1,17 @@
 import { appDataSource } from '../dataSource';
-import { CannotPerformOperationError, NotFoundError } from '../exc';
+import { NotFoundError } from '../exc';
 import { VolunteerRequest, VolunteerRequestToVolunteer } from '../models';
 
 export const volunteerRequestRepository = appDataSource.getRepository(VolunteerRequest).extend({
```

Comment:
I missed the whole point, there should be a new function for managers and coordinators

Reply:
working on it